### ISSUE TITLE
Incomplete display for candidates in multiple current elections

### ISSUE THREAD

**Issue by jf1** [mysociety/yournextrepresentative#799]
Using this example https://candidates.democracyclub.org.uk/person/7452/sian-berry
![image](https://cloud.githubusercontent.com/assets/4711363/14328427/886e4318-fc2e-11e5-9bf7-761bd55f373f.png)
The text "Candidate for Mayor of London in London Mayoral Election" is at best incomplete, or it could be considered inaccurate.

To address this, for candidates standing in multiple current elections the text "Candidate in multiple elections"  could be displayed, with it linking to an anchor (eg of #candidacies) that scrolls the page to:
![image](https://cloud.githubusercontent.com/assets/4711363/14328433/94b605c0-fc2e-11e5-9a41-8ade30be2bfe.png)

**Comment by wfdd** [comment]
Though, when one of the elections and only one of the elections is current, it should show that at the top.

**Comment by jf1** [comment]
Agreed, hence using "multiple current elections" to clarify the scope of this issue.

**Comment by wfdd** [comment]
I should've paid more attention. :)
